src/subsections/landingpage/Banner.js

```diff
@@ -5,12 +5,12 @@ function Banner() {
   return (
     <div className="banner p-4 pt-4 flex items-center flex-col md:flex-row md:justify-center">
       <div className="text-center text-xs text-white opacity-81">
-        Get ready for the HackAPrompt Competition starting May 5th
+        Compete in the HackAPrompt Competition until May 26th!
       </div>
       <a
         target="_blank"
         rel="noreferrer"
-        href={"https://www.aicrowd.com/challenges/hackaprompt-2023"}
```

Comment:
why remove this?

Reply:
Meant to put the link to the competition page - just fixed this in the most recent commit.